CONTRIBUTING.md

```diff
@@ -1,34 +1,45 @@
-# Contributing
+# Contributing to the project
 
 Oracle welcomes contributions to this repository from anyone.
 
 If you want to submit a pull request to fix a bug or enhance an existing feature, please first open an issue and link to that issue when you submit your pull request.
 
 If you have any questions about a possible submission, feel free to open an issue too.
 
-## Contributing to the repository
+## Contributing code
 
-Pull requests can be made under [The Oracle Contributor Agreement](https://oca.opensource.oracle.com/).
+Before submitting code via a pull request, you will need to have signed the [Oracle Contributor Agreement][OCA] (OCA) and your commits need to include the following line using the name and e-mail address you used to sign the OCA:
 
-For pull requests to be accepted, the bottom of your commit message must have the following line using your name and e-mail address as it appears in the OCA Signatories list.
-
-```
+```text
 Signed-off-by: Your Name <you@example.org>
 ```
 
-This can be automatically added to pull requests by committing with:
+This can be automatically added to pull requests by committing with `--sign-off` or `-s`, e.g.
```

Comment:
"add this automatically by committing with..." removes the passive can be and shortens it.